docs/AppService/PerformAppServiceInteraction/index.md

```diff
@@ -16,7 +16,7 @@ Process `serviceUri` according to whatever predefined schema is offered by the H
 !!!
 
 !!! NOTE
-If the HMI is the App Service Consumer (ASC), SDL will:
+If the HMI is the App Service Consumer (<span title="App Service Consumer">ASC</span>), SDL will:
```

Comment:
Do you need to have the span here since you preface the abbreviation with the full name?

Reply:
Removed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/146/commits/d71dde80f9c28f9452c70a2f5b12d551a9505129